opencti-platform/opencti-graphql/src/manager/ingestionManager.ts

```diff
@@ -299,7 +299,10 @@ const csvDataToObjects = async (csvBuffer: Buffer | string, ingestion: BasicStor
 const csvDataHandler = async (context: AuthContext, ingestion: BasicStoreEntityIngestionCsv) => {
   const user = context.user ?? SYSTEM_USER;
   const csvMapper = await findById(context, user, ingestion.csv_mapper_id);
-  const csvMapperParsed = parseCsvMapper(csvMapper);
+  const csvMapperParsed = {
+    ...parseCsvMapper(csvMapper),
+    markings: ingestion.markings ?? []
```

Comment:
`markings` is not declared in `CsvMapperParsed` interface (nor in BasicStoreEntityCsvMapper), why do we add it here ? Is it used afterwards ?

Reply:
Since the CSV feed ingestion entry point to the CSV Import logic is `bundleProcess`, we have opted to pass it like this.
In the CSV import process, the markings are sent in the message. The only thing stored in the database is the configuration (user choice or user default markings).

Markings are retrieve at the beginning of the mapping process.